pkg/cmd/cli/cmd/wrappers.go

```diff
@@ -206,3 +206,59 @@ Examples:
 	cmd.Long = fmt.Sprintf(longDesc, fullName)
 	return cmd
 }
+
+// NewCmdStop is a wrapper for the Kubernetes cli stop command
+func NewCmdStop(fullName string, f *clientcmd.Factory, out io.Writer) *cobra.Command {
+	cmd := cmd.NewCmdStop(f.Factory, out)
+	longDesc := `Gracefully shut down a resource by id or filename.
+
+Attempts to shut down and delete a resource that supports graceful termination.
+If the resource is resizable it will be resized to 0 before deletion.
+
+Examples:
+
+	# Shut down foo.
+	$ %[1]s stop replicationcontroller foo
+
+	# Stop pods and services with label name=myLabel.
+	$ %[1]s stop pods,services -l name=myLabel
+
+	# Shut down the service defined in service.json
+	$ %[1]s stop -f service.json
+
+	# Shut down all resources in the path/to/resources directory
+	$ %[1]s stop -f path/to/resources
+`
+	cmd.Long = fmt.Sprintf(longDesc, fullName)
```

Comment:
----- Original Message -----
> > +
> > +Examples:
> > +
> > +	# Shut down foo.
> > +	$ %[1]s stop replicationcontroller foo
> > +
> > +	# Stop pods and services with label name=myLabel.
> > +	$ %[1]s stop pods,services -l name=myLabel
> > +
> > +	# Shut down the service defined in service.json
> > +	$ %[1]s stop -f service.json
> > +
> > +	# Shut down all resources in the path/to/resources directory
> > +	$ %[1]s stop -f path/to/resources
> > +`
> > +	cmd.Long = fmt.Sprintf(longDesc, fullName)
> 
> Well, maybe I'm wrong about the resizer but still...

Resizer I believe has an abstraction as well that goes into the Factory.  There are other issues with Resize though, which is that currently deployments aren't preserving resize and it's not clear whether the deployment configs value for replicas should be used.  For now, Resizer resizing the current dcs/deployment would be most correct.